docs/SQL/development/development.md

```diff
@@ -56,12 +42,39 @@ Vast majority of dbt and SQL constructs are supported within the Prophecy Visual
+
+## Deployment from code on Git
+
+Projects built through Prophecy are stored in the dbt Core format as repositories on Git, which allows data teams to follow best software engineering practices like CI/CD.
+
+Maintenance is simple since Prophecy gems turn into code on Git that’s always up-to-date with the latest version of the warehouse or lakehouse used. And, to ensure the best performance at all times, Prophecy is smart about which code construct (subquery or CTE) to use.
+
+## Sharing of Projects as Packages
+
+Data users can import an existing dbt Core project or start from scratch. They can publish those projects to other teams or subscribe to already existing ones. Projects published as packages contain models, functions and Gems allowing for code reusability at every level.
+
+## Learn more
+
+A word from Prophecy's co-Founder, Maciej! See how Prophecy allows every team, whether visual or code developers, to use the same software development best practices.
+
+<div class="wistia_responsive_padding" style={{padding:'56% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/3i9g2dciah?seo=false?videoFoam=true" title="Design a Pipeline Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
 ## Hands-on
 
-To get started developing SQL models, check out the pages below or try the getting-started [guide](/docs/getting-started/getting-started-with-low-code-sql.md).
+To get started developing SQL models, check out the pages below or try this guide for [Databricks](/docs/getting-started/getting-started-with-low-code-sql.md) or [Snowflake](/docs/getting-started/getting-started-sql-snowflake.md).
 
-1. [**Models**](./models.md) - All the steps needed to define a table or view.
+```mdx-code-block
+import DocCardList from '@theme/DocCardList';
+import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
 
-2. [**Gems**](./gems/gems.md) - The individual steps, or transformations, to define a table or view.
+<DocCardList items={useCurrentSidebarCategory().items}/>
+```
 
-As you're building your Model by adding individual Gems, see how the process is going by interactively running the model. Explore the [Interactive Development](./interactive-development/data-explorer.md) page to learn more.
+As you're building your Model by adding individual Gems, see how the process is going by interactively running the model. To learn more, explore the [Data Explorer](visual-editor/interactive-development/data-explorer) page under Interactive Development.
```

Comment:
Nit: "individual Gems" can link to /SQL/gems/

Reply:
Done.